### DISCUSSION TITLE
GLSP vscode client: How we configure color of Manual and Automation Node

### DISCUSSION THREAD

**Discussion by sathya-1994** [eclipse-glsp/glsp#396] [General] [answered]
I am using GLSP's vscode client(https://github.com/eclipse-glsp/glsp-vscode-integration)

I want change color of automated node. In the generated graph its color is 'gray'. But this is not specified anywhere in the GLSP's vscode client project. But generated webview.js has 'gray' color entry. How we configure colors of manual and automated node using css ?

![image](https://user-images.githubusercontent.com/59145404/133560076-461be7bf-f432-4919-a593-93ac4db9cc7a.png)
![image](https://user-images.githubusercontent.com/59145404/133561439-7672eba8-5898-41e1-bc24-c97ba2725819.png)

**Comment by planger** [comment, marked as answer]
The CSS you are referring to is coming from the Workflow example of the [GLSP client repository.](https://github.com/eclipse-glsp/glsp-client)

This package is consumed by the webview here:
https://github.com/eclipse-glsp/glsp-vscode-integration/blob/7e40cf763ff59658e5bf6467e403416b3d04f293/example/workflow/webview/package.json#L32

The actual source code of the consumed package is here:
https://github.com/eclipse-glsp/glsp-client/blob/9793bf3179a08b8808e9ba0c76adbe413bbba97b/examples/workflow-glsp/css/diagram.css#L123

The Workflow example is just an example diagram that is common to all GLSP packages (client, server, Theia integration, VSCode integration). Typically you would just use this example as a blueprint, but create your own packages with your own adaptations for your diagram editor and use/publish/consume your packages instead of using the workflow packages directly.
